common/djangoapps/student/tests/test_refunds.py

```diff
@@ -96,34 +97,39 @@ def test_refundable_of_purchased_course(self):
 
     def test_refundable_when_certificate_exists(self):
         """ Assert that enrollment is not refundable once a certificat has been generated."""
-        self.assertTrue(self.enrollment.refundable())
+        with patch('student.models.CourseEnrollment.refund_cutoff_date') as cutoff_date:
```

Comment:
this pattern appears a lot; you'll see fewer diffs (and fewer indents) if you use the patch decorator rather than the context manager